src/ui/public/vis/agg_configs.js

```diff
@@ -145,14 +144,45 @@ export default function AggConfigsFactory(Private) {
     });
 
     removeParentAggs(dslTopLvl);
-
     return dslTopLvl;
   };
 
   AggConfigs.prototype.getRequestAggs = function () {
-    return _.sortBy(this, function (agg) {
+
+
+    const aggregations = _.sortBy(this, function (agg) {
       return agg.schema.group === 'metrics' ? 1 : 0;
     });
+
+
+    let shouldAddGeocentroid = false;
```

Comment:
I sort of agree. It is not the best place to put it.

Geo-centroid is a separate metric in ES, but we do not want this to expose as such in Kibana-Visualize. Otherwise, people would need to explicitly select this. 

IMO, any geohash-aggregation should include the geocentroid-metric, hence I added this here.

Any thoughts?